### PR TITLE
FreeBSD support

### PR DESCRIPTION
**Pull request by rminnich**

### CODE REVIEW

File: cmds/cpuns/main_unix.go [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+	v("CPUNS:"+f, a...)
+}
+
+func checkprivate() error {
+	mnts, err := mountinfo.GetMounts(mountinfo.SingleEntryFilter("/"))
+	if err != nil {
+		return err
+	}
+	if len(mnts) != 1 {
+		return fmt.Errorf("got more than 1 mount for /(%v):%v", mnts, os.ErrInvalid)
```
Comment (binjip978) [inline cmds/cpuns/main_unix.go]: nit: %w for os.ErrInvalid

File: mount/mount_freebsd.go [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+		}
+		if w, err := filepath.EvalSymlinks(dev); err == nil {
+			dev = w
+		}
+
+		// The man page implies that the Linux kernel handles flags of "defaults"
+		// we do no further manipulation of opts.
+		flags, data := parse(opts)
+		if e := m(dev, where, fstype, flags, data); e != nil {
+			err = errors.Join(err, fmt.Errorf("Mount(%q, %q, %q, %q=>(%#x, %q)): %v", dev, where, fstype, opts, flags, data, e))
```
Comment (binjip978) [inline mount/mount_freebsd.go]: nit: %w for error

File: mount/mount_linux.go [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+		}
+		if w, err := filepath.EvalSymlinks(dev); err == nil {
+			dev = w
+		}
+
+		// The man page implies that the Linux kernel handles flags of "defaults"
+		// we do no further manipulation of opts.
+		flags, data := parse(opts)
+		if e := m(dev, where, fstype, flags, data); e != nil {
+			err = errors.Join(err, fmt.Errorf("Mount(%q, %q, %q, %q=>(%#x, %q)): %v", dev, where, fstype, opts, flags, data, e))
```
Comment (binjip978) [inline mount/mount_linux.go]: nit: %w for error

**Review by binjip978 — APPROVED** [review]

File: cmds/cpuns/main_unix.go (hunk 1 above)
Comment (rminnich) [reply cmds/cpuns/main_unix.go]: done

File: mount/mount_linux.go (hunk 3 above)
Comment (rminnich) [reply mount/mount_linux.go]: done

File: mount/mount_freebsd.go (hunk 2 above)
Comment (rminnich) [reply mount/mount_freebsd.go]: done

**Review by binjip978 — APPROVED** [review]

**Pull request merged**